api/test/integration/test_rbac_white_agent_endpoints.tavern.yaml

```diff
@@ -617,16 +581,6 @@ stages:
        fields: os.platform
    response:
      status_code: 200
-     json:
-       error: 0
-       data:
-         affected_items:
-           - count: 6
-             os:
-               platform: !anystr
-         failed_items: []
-         total_affected_items: !anyint
-         total_failed_items: 0
```

Comment:
As you can read in the test name, this test permissions are partially allowed (we are asking for for distinct stats in all agents). This means that the `count` field needs to be `6` exactly, as we have read permissions for 6 agents in this environment.